### PR TITLE
A0-2998: Put 'true' in double quote in one the conditions to check if that helps

### PR DESCRIPTION
**Pull request by MikolajGasior**
# Description
Even though input as a boolean, its value in condition is cast to string.

### CODE REVIEW

**Review by CezaryKierzyk — APPROVED** [review]

File: .github/workflows/_check-vars-and-secrets.yml [hunk 1]
```diff
@@ -21,9 +26,32 @@ jobs:
     name: Check available vars and secrets
     runs-on: ubuntu-20.04
     steps:
+      - name: Debug input values
```
Comment (timorl) [inline .github/workflows/_check-vars-and-secrets.yml:29]: This shouldn't stay here, right?

**Review by timorl — APPROVED** [review]

File: .github/workflows/_check-vars-and-secrets.yml [hunk 2]
```diff
@@ -21,9 +26,32 @@ jobs:
...
-        if: inputs.check-vars == true
         run: |
+          if [[ "${{ inputs.check-vars }}" != "true" ]]; then
+            echo "Skipping vars check"
+            exit 0
+          fi
+
           if [[ \
             -z "${{ vars.ECR_PUBLIC_HOST }}" || \
             -z "${{ vars.ECR_PUBLIC_REGISTRY }}" || \
```
Comment (ggawryal) [inline .github/workflows/_check-vars-and-secrets.yml:57]: Do we want to check `"${{ vars.RUST_TOOLCHAIN_NIGHTLY }}"` in both `dev-vars` and `vars`?

**Review by ggawryal — APPROVED** [review]

File: .github/workflows/_check-vars-and-secrets.yml (hunk 1 above)
Comment (MikolajGasior) [reply .github/workflows/_check-vars-and-secrets.yml]: Removed

File: .github/workflows/_check-vars-and-secrets.yml (hunk 2 above)
Comment (MikolajGasior) [reply .github/workflows/_check-vars-and-secrets.yml:53]: Oh boy, yes, of course!  It's removed now.

**Pull request merged**